### PR TITLE
i18n: updated Italian translations

### PR DESCRIPTION
**Pull request by nitefood**
Updated italian translations

#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [ ] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 

#### Description

 - changed some translations for clarity
 - translated most recently added strings
 - fixed some typos

### CODE REVIEW

**Comment by Kaian** [comment]
Hi @nitefood 

I have little idea about italian, but test passed so It's ok to me.

Maybe @dsmaldone (original translator) is willing to check the changes. I'll leave this open a couple days in case some commentary appears.

Thanks for your contribution!!

**Comment by dsmaldone** [comment]
HI @Kaian 
I will do this with pleasure!
Thx @nitefood

**Comment by Kaian** [comment]
Hi @nitefood 

@dsmaldone have reviewed the translations in #1482 (https://github.com/irontec/ivozprovider/pull/1482/commits/41a99df9896615c3a840005e30ba1d22b2e275a4) 

You can download the raw version of the translation file and update the PR with any changes you want to apply.
I recommend rebasing your branch from latest upstream bleeding because some strings may have changed/deleted/created since the PR was opened.

Don't hesitate to request assistance for this process if you need it.

Thanks both of you for your work in Italian translations!

**Comment by Kaian** [comment]
> Can't see changes in #1427 seems like they are still in #1482

Yep, @nitefood must apply them whenever he can. Let's give him some time :)

File: web/admin/application/languages/it_IT/it_IT.po [hunk 1]
```diff
@@ -2815,7 +2817,7 @@ msgstr "Tempo in secondi per cui squilleranno i membri della coda "
 msgid ""
 "Time in seconds that member won't be disturbed after attending a queue call"
 msgstr ""
-"Tempo in secondi per cui un membro non sarà contattato dopo aver partecipato "
+"Tempo in secondi per cui un membro non sarà disturbato dopo aver partecipato "
```
[the remark this answers is not in the document]
Comment (nitefood) [reply web/admin/application/languages/it_IT/it_IT.po:2820]: I completely agree, "contattato" sounds much better

File: web/admin/application/languages/it_IT/it_IT.po [hunk 2]
```diff
@@ -1727,13 +1729,13 @@ msgid "Matching type"
 msgstr "Confronta tipo"
 
 msgid "Max calls"
-msgstr "Massimo Chiamates"
```
[the remark this answers is not in the document]
Comment (nitefood) [reply web/admin/application/languages/it_IT/it_IT.po:1730]: I don't agree here, "Massimo Chiamate" doesn't quite sound right. I mean, you're right in that "Numero Massimo di Chiamate" would be a better translation, but that's (IMHO) too long in comparison to the original string.
How about "Chiamate Max" or "Max Chiamate"? Sounds to me like a good compromise.

[the remark this answers is not in the document]
Comment (dsmaldone) [reply web/admin/application/languages/it_IT/it_IT.po:1740]: Looks ok to me "Max Chiamate"

**Comment by Kaian** [comment]
Looks good to me, I'll be merging this today.

Thanks for your hard work to both of you!

**Comment by dsmaldone** [comment]
Now I can see all the work we did and I'm happy!
I can finally see all the efforts made in commit 1bbe7278a14f5435ff48bf303d335d665f61d017

**Pull request merged**